odoo/fields.py

```diff
@@ -34,6 +34,7 @@
 EMPTY_DICT = frozendict()
 
 RENAMED_ATTRS = [('select', 'index'), ('digits_compute', 'digits')]
+DEPRECATED_ATTRS = [("oldname", "use a upgrade script instead.")]
```

Comment:
AFAIK there's no existing "upgrade script" in the odoo lingo. Shouldn't that be a "migration script"? 

And maybe ping whoever's in charge of the migration team for input?

Reply:
Yes, it is the same as "migration script", but using a better term.

The term "upgrade" is already used for the service (upgrade.odoo.com) and we will use
it when #32650 will be merged

Reply:
Technically, in English the term upgrade is more correct than migration since an upgrade is switching to a newer version of the software (and that might include modifying the database schema and data) whereas a migration is more often used to describe a complete switch to another software. 

This is highly debatable though and while I'm personally in favour of _upgrades_, I'm sure that the Odoo lingo can get used to both if need be.

Reply:
Works for me, I just hadn't seen #32650 and been made aware that we're changing the lingo. And checking the repo it's still all "migration".